### PR TITLE
[DW-2809] Removed p tag from table

### PR DESCRIPTION
**Pull request by Deven-Rawat**

### CODE REVIEW

File: site/components/src/test/java/uk/nhs/digital/common/contentrewriters/BrandRefreshContentRewriterTest.java [hunk 1]
```diff
@@ -281,7 +281,7 @@ public void rewriteTestForTableWithoutHeader() {
         assertTrue(result.contains("p"));
         assertTrue(result.contains("table"));
         assertFalse(result.contains("cannotsort"));
-        assertFalse(result.contains("caption")); // caption tag removed and text placed in heading
+        assertTrue(result.contains("caption")); // caption tag removed and text placed in heading
```
Comment (LEJA3) [inline site/components/src/test/java/uk/nhs/digital/common/contentrewriters/BrandRefreshContentRewriterTest.java:284]: Could we remove the comment here now that it no longer applies?

File: site/components/src/test/java/uk/nhs/digital/common/contentrewriters/BrandRefreshContentRewriterTest.java [hunk 2]
```diff
@@ -319,7 +317,7 @@ public void rewriteTestForTable() {
...
```
Comment (LEJA3) [inline site/components/src/test/java/uk/nhs/digital/common/contentrewriters/BrandRefreshContentRewriterTest.java:320]: Same here

File: site/components/src/test/java/uk/nhs/digital/common/contentrewriters/BrandRefreshContentRewriterTest.java (hunk 1 above)
Comment (Deven-Rawat) [reply site/components/src/test/java/uk/nhs/digital/common/contentrewriters/BrandRefreshContentRewriterTest.java]: removed

File: site/components/src/test/java/uk/nhs/digital/common/contentrewriters/BrandRefreshContentRewriterTest.java (hunk 2 above)
Comment (Deven-Rawat) [reply site/components/src/test/java/uk/nhs/digital/common/contentrewriters/BrandRefreshContentRewriterTest.java]: removed

**Review by LEJA3 — APPROVED** [review]

**Pull request merged**